SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -84,6 +88,21 @@ - (void)sdl_stopEventListening {
     [SDLDebugTool logInfo:@"SDLIAPTransport Stopped Listening For Events"];
     [[NSNotificationCenter defaultCenter] removeObserver:self];
 }
+- (void)sdl_backgroundTaskStart {
+    if (self.backgroundTaskId == UIBackgroundTaskInvalid) {
```

Comment:
This should go the other way, `self.backgroundTaskId != UIBackgroundTaskInvalid`, and return early.